### PR TITLE
update linkerd zipkin to 1.6.1

### PR DESCRIPTION
**Pull request by dadjeibaah**
Signed-off-by: Dennis Adjei-Baah <dennis@buoyant.io>

### CODE REVIEW

**Review by klingerf — APPROVED** [review]
⭐️ Looks good, thanks for updating!

File: build.sbt [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 def finagle(mod: String) =
-  "com.twitter" %% s"finagle-$mod" % "18.9.1"
+  "com.twitter" %% s"finagle-$mod" % "18.12.0"
 
 def telemetry(mod: String) =
   "io.buoyant" %% s"telemetry-$mod" % linkerdVersion
 
 def zipkin(mod: String) =
-  "io.zipkin.finagle2" %% s"zipkin-finagle-$mod" % "2.0.5"
+  "io.zipkin.finagle2" %% s"zipkin-finagle-$mod" % "2.0.8"
```
Comment (codefromthecrypt) [inline build.sbt:15]: 2.0.11 was actually the current rev, fyi

Comment (codefromthecrypt) [reply build.sbt:15]: whoops nevermind.. the finagle versions aren't latest here!

Comment (dadjeibaah) [reply build.sbt:15]: Yea, I wanted to use the latest version but decided that it is probably best to make sure all the finagle version aligns with finagle-zipkin's version.

**Pull request merged**